config.json

```diff
@@ -34,7 +34,7 @@
       "description": "Specifies the distro rid for Unix OS.",
       "valueType": "property",
       "values": [],
-      "defaultValue": "${OSRid}"
+      "defaultValue": "${OSRid}-${CPUArch}"
```

Comment:
At some point I want to switch away from this DistroRID and use something like RuntimeOS (https://github.com/dotnet/corefx/blob/master/config.json#L33) like we did in corefx. That gets generated as a configuration when doing the build so that we have a consistent way of getting the OS without crazy parsing in various targets/projects.